### DISCUSSION TITLE
Performance measurements of vinyl engine

### DISCUSSION THREAD

**Discussion by kyukhin** [tarantool/tarantool#5639] [Ideas]
Sometimes I hear questions about performance of vinyl and the only answer I can provide is that it writes fast and reads not that fast.

Probably it'll be useful to have a comparison table with follogwing dimensions:
1. Different HW: tmpfs, SSD, HDD, RAID*
2. Different DBMS: Tarantool (versions?), RocksDB, MariaDB, Postgres
3. Different workload profiles: more reads, more writes, long transactions, short transactions (I mean long running or not), big entries, small entries etc
4. Different access patterns and schema: secondary keys, unique fields, different indexes
5. Type of data access: point selects, range selects etc.
6. Different wal modes

I think there might be more dimensions.

**Comment by jobs-git** [comment]
I will try to work MariaDB vs Tarantool, do you have specifics so that there is a standardized test across all other DBs? I only have one hardware to test though.

**Comment by filonenko-mikhail** [comment]
Want to add MongoDB in comparison list
